src/utils.js

```diff
@@ -87,21 +87,34 @@ export async function asyncBufferFromUrl({ url, byteLength, requestInit }) {
 /**
  * Construct an AsyncBuffer for a local file using node fs package.
  *
+ * The function is not supported in the browser.
+ *
  * @param {string} filename
  * @returns {Promise<AsyncBuffer>}
  */
 export async function asyncBufferFromFile(filename) {
-  const fsPackage = 'fs' // webpack no include
```

Comment:
I think this PR missed the reason for the "webpack no include" hack. If you put `await import('fs')` then webpack looks at it and says "oh it looks like I need to bundle this too" but then fails because `fs` is a node system package not available in npm or the browser.

So the problem is not exception handling, it's the bundling.

I _think_ that the solution is to have a conditional export. Possible that `asyncBufferFromFile` needs to be in its own file to avoid aggressive bundling, but not sure.
https://nodejs.org/api/packages.html#conditional-exports

I'll see if I can get a repro on the bundling issue. I think it happened when I imported from hyperparam or the cli without the string hack.

Reply:
As mentioned in the PR message, I'm not sure if conditional exports is the solution because a Node.js app can use either ESM imports or CJS require.